### PR TITLE
Add OWASP Maryam

### PR DESCRIPTION
**Pull request by saeeddhqan**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ permalink: /initiatives/gsoc/gsoc2023ideas
 
 [ZAP](#owaspzap) | [Bug Logging Tool (BLT)](#bug-logging-tool-blt)
```
Comment (psiinon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:10]: You need to add Maryam here 😁

Comment (saeeddhqan) [reply pages/initiatives/gsoc/gsoc2023ideas.md:10]: :) Is that correct now?

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ permalink: /initiatives/gsoc/gsoc2023ideas
...
 
+[Maryam](#owaspmaryam)
+
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:13]:
```suggestion
[ZAP](#owaspzap) | [Bug Logging Tool (BLT)](#bug-logging-tool-blt) | [Maryam](#owaspmaryam)

```

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 3]
```diff
@@ -132,3 +134,52 @@ Reach out to us on Slack to discuss further on the scope, changes required, or i
 * Rahul Badami - Payments Mentor  
 * [Ankit Choudhary](mailto:ankitchoudhary202.ac@gmail.com) - Flutter Mentor  
 * [Sparsh Agrawal](mailto:sparshagrawal1212@gmail.com) - Flutter Mentor
+
+
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:138]:
```suggestion
```

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 4]
```diff
@@ -132,3 +134,52 @@ Reach out to us on Slack to discuss further on the scope, changes required, or i
...
+
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:139]:
```suggestion

```

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 5]
```diff
@@ -132,3 +132,49 @@ Reach out to us on Slack to discuss further on the scope, changes required, or i
 * Rahul Badami - Payments Mentor  
 * [Ankit Choudhary](mailto:ankitchoudhary202.ac@gmail.com) - Flutter Mentor  
 * [Sparsh Agrawal](mailto:sparshagrawal1212@gmail.com) - Flutter Mentor
+### [OWASP Maryam](https://owasp.org/www-project-maryam/)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:135]:
```suggestion

### [OWASP Maryam](https://owasp.org/www-project-maryam/)
```

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 6]
```diff
@@ -132,3 +132,50 @@ Reach out to us on Slack to discuss further on the scope, changes required, or i
...
+
+If you have a new Idea regarding Iris(Meta searcher), let us chat.
+
+##### Getting Started
+
+* [OWASP Maryam](https://github.com/saeeddhqan/Maryam)
+* [Documentation](https://github.com/saeeddhqan/Maryam/wiki)
+
+
+##### Mentors
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:179]:
```suggestion
* [Documentation](https://github.com/saeeddhqan/Maryam/wiki)

##### Mentors
```

**Review by kingthorin — APPROVED** [review]

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 4 above)
Comment (vico1973) [reply pages/initiatives/gsoc/gsoc2023ideas.md]: >

**Pull request merged**